src/CMakeLists.txt

```diff
@@ -462,6 +466,10 @@ ADD_SUBDIRECTORY ( Commands )
 ADD_SUBDIRECTORY ( Desktop )
 ADD_SUBDIRECTORY ( CommandLine )
 ADD_SUBDIRECTORY ( Tests )
+if (CMAKE_COMPILER_IS_GNUCC)
```

Comment:
FYI, our windows builds use the MSVC compiler currently.  We don't need these speedups to also be in the windows build (our windows userbase is small, partly because fsl and freesurfer don't support windows at all), but do you know how much work it would be?

Also, we might want to try clang at some point (doesn't it share a lot of front end with gcc?), so some perspective there would also be useful for future reference.